node/src/sub_lib/utils.rs

```diff
@@ -251,6 +258,41 @@ pub struct MessageScheduler<M: Message> {
     pub delay: Duration,
 }
 
+#[cfg(target_os = "windows")]
+pub fn wsa_startup_init() {
+    let lp_vendor: *mut u8 = 1 as *mut u8;
+    let wsdata: *mut windows_sys::Win32::Networking::WinSock::WSADATA =
+        &mut windows_sys::Win32::Networking::WinSock::WSADATA {
+            wVersion: 2.2 as u16,
+            wHighVersion: 2.2 as u16,
+            iMaxSockets: 0,
+            iMaxUdpDg: win_cfg::SO_MAX_MSG_SIZE as u16,
+            lpVendorInfo: lp_vendor as win_cfg::PSTR,
```

Comment:
These three elements are all obsolete. Could you set them all to zero and provide `// obsolete` comments (or something equally intention-revealing) to alleviate future concerns about values?

Also, that will let you get rid of `lp_vendor` as a mutable pointer to absolute address 0x0000000000000001, which is a bit alarming.